openmdao/recorders/tests/test_sqlite_recorder.py

```diff
@@ -2588,7 +2589,12 @@ def setup(self):
         prob['circuit.n1.V'] = 10.
         prob['circuit.n2.V'] = 1.
 
-        prob.run_driver()
+        msg = ("BoundsEnforceLS in Circuit (circuit): linesearch is active but no bounds have been "
+               "set.")
+
+        with assert_no_warning(UserWarning, msg):
+            prob.run_driver()
```

Comment:
It seems odd to check for a very specific warning that doesn't exist anywhere in the code base...
Maybe we just want a version of this assert that checks for no warnings at all?

Reply:
Wasn't there a problem where other warnings sometimes show up depending on changes we make in future stories (or sometimes temporarily due to updates in other packages like numpy) which would cause this to fail because it expects no warning?

Reply:
You make a good point though, that maybe we should just remove the test.

Reply:
Agree... seems like just removing the assert would be appropriate

Reply:
Agree.

Reply:
Removed